pyscriptjs/tests/integration/test_01_basic.py

```diff
@@ -330,6 +330,41 @@ def myfunc():
         )
         assert banner.inner_text() == expected_message
 
+    def test_py_click(self):
+        self.pyscript_run(
+            """
+            <button id="btn" py-click="myfunc(js_event)">Click me</button>
```

Comment:
does this need to be called `js_event` instead just `event` ? I am asking because by default that's what HTML passes along (it's global) on listeners:

```html
<!doctype html>
<script>
function myfn(event) {
  console.log(event.type, event.currentTarget);
}
</script>
<div onclick="myfn(event)">
  Click Me
</div>
```